SDCAlertView UI Tests/SDCAlertView_UI_Tests.swift

```diff
@@ -15,20 +15,28 @@ class SDCAlertView_UI_Tests: XCTestCase {
 
     private func currentAlert() -> XCUIElement {
         return XCUIApplication().children(matching: .window)
-                                .elementBound(by: 0)
+                                .element(boundBy: 0)
                                 .children(matching: .other)
-                                .elementBound(by: 1)
+                                .element(boundBy: 1)
                                 .children(matching: .other)
-                                .elementBound(by: 1)
+                                .element(boundBy: 1)
     }
 
     private func buttonAtIndex(_ index: UInt) -> XCUIElement {
-        return XCUIApplication().collectionViews.children(matching: .cell).elementBound(by: index)
+        return XCUIApplication().collectionViews.children(matching: .cell).element(boundBy: index)
     }
 
-    private func showAlertAtIndex(_ index: UInt) -> XCUIApplication {
+    private func buttonWithIdentifier(_ identifier: String) -> XCUIElement {
+        let cell = XCUIApplication().collectionViews.cells.matching(identifier: identifier).element(boundBy: 0)
+        if !cell.exists {
+            return XCUIApplication().buttons.matching(identifier: identifier).element(boundBy: 0)
+        }
+        return cell
+    }
+
+    @discardableResult private func showAlertAtIndex(_ index: UInt) -> XCUIApplication {
```

Comment:
Style: please add a line break after `@discardableResult` (so that `private func` starts on a new line)